### ISSUE TITLE
Error: stream encoding should not be set

### ISSUE THREAD

**Issue by markspolakovs** [callstack-io/haul#136]
### Current Behavior

When tapping a stack trace in the app (in a RedBox or YellowBox), the Haul console prints the following error:

```
Error: stream encoding should not be set
    at readStream (D:\Code\euroschool-app\node_modules\raw-body\index.js:211:17)
    at getRawBody (D:\Code\euroschool-app\node_modules\raw-body\index.js:106:12)
    at read (D:\Code\euroschool-app\node_modules\body-parser\lib\read.js:76:3)
    at textParser (D:\Code\euroschool-app\node_modules\body-parser\lib\types\text.js:86:5)
    at Layer.handle [as handle_request] (D:\Code\euroschool-app\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Code\euroschool-app\node_modules\express\lib\router\index.js:317:13)
    at D:\Code\euroschool-app\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (D:\Code\euroschool-app\node_modules\express\lib\router\index.js:335:12)
    at next (D:\Code\euroschool-app\node_modules\express\lib\router\index.js:275:10)
    at IncomingMessage.req.on (D:/Code/euroschool-app/node_modules/haul-cli/src/server/middleware/rawBodyMiddleware.js:16:5)
```

### Haul Configuration (webpack.haul.js)

```js
module.exports = ({ platform }, { module }) => ({
    entry: `./index.${platform}.js`,
    module: {
        ...module,
        rules: [
            {
                test: /\.tsx?$/,
                loader: 'awesome-typescript-loader',
            },
            ...module.rules,
        ],
    },
});
```

### Your Environment

Running on Windows 10 Creators Update (10.0.15063)

| software         | version |
| ---------------- | ------- |
| Haul             |  0.5.0 |
| react-native     |  0.43.4 |
| node             | 6.10.0 |
| npm or yarn      | 0.23.2 |

**Comment by grabbou** [comment]
Is this something that could be possibly fix by #123 ?

**Comment by markspolakovs** [comment]
@grabbou I doubt it. Even though file opening is NYI, it shouldn't throw an encoding error.

**Comment by skellock** [comment]
This was fixed by #124.  It's in master and ready to go for next release.

**Comment by satya164** [comment]
Cool. Closing this then. Let's do a new release today @grabbou
